pkg/apis/scorecard/v1alpha2/formatter.go

```diff
@@ -46,27 +46,37 @@ func (s ScorecardOutput) MarshalText() (string, error) {
 			sb.WriteString(fmt.Sprintf("%s:\n", result.Labels["suite"]))
 			currentSuite = result.Labels["suite"]
 		}
-		sb.WriteString(fmt.Sprintf("\t%-35s: ", result.Name))
+		sb.WriteString(fmt.Sprintf("\t%-35s ", result.Name))
 
 		if result.State == PassState {
 			sb.WriteString(fmt.Sprintf(passColor, PassState))
-		} else {
+		} else if result.State == FailState {
 			sb.WriteString(fmt.Sprintf(failColor, FailState))
+		} else {
+			sb.WriteString(fmt.Sprintf("\n"))
 		}
-	}
 
-	for _, result := range s.Results {
-		for _, suggestion := range result.Suggestions {
+		sb.WriteString("\tLabels: \n")
```

Comment:
Now that we're printing the labels in the text output, should we remove the 0th tab level suite grouping since the suite is in a label? Perhaps the 0th level should be the test name and the rest of the test result fields (Labels, Suggestions, Errors, etc.) go on the 1st tab level so they're one level indented from the test name?

I know this formatting stuff is a bit out of scope for this PR, so we can make these changes in a follow-up if you'd like.

Reply:
yep, lets do these formatting changes under the output format story if thats ok with everyone.   Lots of different output formats for sure to consider.